package.json

```
@@ -4,7 +4,9 @@
   "description": "",
   "main": "bot.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "tslint --project .",
+    "postinstall": "tsc --project .",
```

```suggestion
    "postinstall": "tsc --project . --outdir ./build",
```

Add a build directory to make things clean